Ignore/CSS Issue Tracker background Login Page.css

```diff
@@ -0,0 +1,15 @@
+body {
```

Comment:
In the interest of keeping the code clean, let's not commit anything that shouldn't be part of the project.